app/routes/application.js

```diff
@@ -8,10 +10,10 @@ export default class ApplicationRoute extends Route {
 
   async beforeModel() {
     await this.session.setup();
-    const intl = this.intl;
-    const moment = this.moment;
-    moment.setLocale(intl.locale);
-    window.document.querySelector('html').setAttribute('lang', intl.locale);
+    await loadPolyfills();
+    const locale = this.intl.get('primaryLocale');
+    this.moment.setLocale(locale);
+    window.document.querySelector('html').setAttribute('lang', locale);
```

Comment:
This should be omitted in the LTI as we don't control the HTML here.

Reply:
:heavy_check_mark: